documentation/library_guide/parallel_api/execution_policies.rst

```diff
@@ -1,65 +1,70 @@
 Execution Policies
 ##################
 
-The implementation supports the device execution policies used to run the massive parallel
-computational model for heterogeneous systems. The policies are specified in
-the |onedpl_long| (|onedpl_short|) section of the `oneAPI Specification
-<https://spec.oneapi.io/versions/latest/elements/oneDPL/source/parallel_api.html#dpc-execution-policy>`_.
-
-For any of the implemented algorithms, pass one of the execution policy objects as the first
-argument in a call to specify the desired execution behavior. The policies have
-the following meaning:
-
-================================= ==============================
-Execution Policy Value            Description
-================================= ==============================
-``seq``                           Sequential execution.
---------------------------------- ------------------------------
-``unseq``                         Unsequenced SIMD execution. This policy requires that
-                                  all functions provided are SIMD-safe.
---------------------------------- ------------------------------
-``par``                           Parallel execution by multiple threads.
---------------------------------- ------------------------------
-``par_unseq``                     Combined effect of ``unseq`` and ``par``.
---------------------------------- ------------------------------
-``dpcpp_default``                 Massive parallel execution on devices using |dpcpp_short|.
---------------------------------- ------------------------------
-``dpcpp_fpga``                    Massive parallel execution on FPGA devices.
-================================= ==============================
+According to `the oneAPI specification
+<https://uxlfoundation.github.io/oneAPI-spec/spec/elements/oneDPL/source/index.html>`_,
+|onedpl_long| (|onedpl_short|) provides execution policies semantically aligned with the C++ standard,
+also referred to as *standard aligned* or *host execution policies*, and the *device execution policies*
+to run data parallel computations on heterogeneous systems.
+
+The execution policies are defined in the ``oneapi::dpl::execution`` namespace and provided
+in the ``oneapi/dpl/execution`` header. The policies have the following meaning:
+
+====================== =====================================================
+Policy Value or Type   Description
```

Comment:
Probably, does it makes sense to "split" (to have two tables) for the host policies
(policy values only) and for device policy types and policy values.
Otherwise, the column "Policy Value or Type" might confuse a user.

Reply:
Thanks.
The description indicates what is type and what is a value. I can also add "type" to the types :)

Reply:
Tried a few variations, and ended up with changing the column heading to "Policy Name / Type".